### ISSUE TITLE
Fasta doesn't emit error or warning for field overriden by method

### ISSUE THREAD

**Issue by sjindel-google** [dart-lang/sdk#31415]
E.g.:
```
class C {
var foo;
}

class D extends C {
void foo() {}
}
```

No error or warning from Fasta.
Also the reverse is allowed:
```
class C {
void foo() {};
}
class D extends C {
var foo;
}
```
No error or warning here either.

**Comment by vsmenon** [comment]
@kmillikin - I bumped this to Beta 3.  Can you move to [Beta 4 ](https://github.com/dart-lang/sdk/milestone/54)or [Dart 2](https://github.com/dart-lang/sdk/milestone/12) if one of those is more appropriate?

**Comment by kmillikin** [comment]
We have not implemented (all) override detection yet, so this is an instance of the broader issue.  I'm moving it to the next mileston (Dart2 stable) where all the other issues with incorrect code are going.

**Comment by kmillikin** [comment]
Duplicate of issue #31850.  Fixed in 8f6c6c2a8edd514cdb6533541189a6572318facd.
